screenpy/__init__.py

```diff
@@ -77,7 +77,6 @@
     "QuestionError",
     "scene",
     "ScreenPyError",
-    "the_narrator",
```

Comment:
Folks can still do `from screenpy.pacing import the_narrator` if they like, but now we don't have the confusion i was worried about.